### PR TITLE
Fix `commandline --cursor-end`

### PR DESCRIPTION
**Pull request by abusch**
# Description

In `commandline --cursor-end`, set `repl.cursor_pos` to the number of bytes in the buffer, not the number of graphemes.

fixes: #11503

# User-Facing Changes

# Tests + Formatting

# After Submitting

### CODE REVIEW

**Review by fdncred — APPROVED** [review]
We have a release scheduled for tomorrow, so let's wait until after that release to land this. I think it looks good though.

**Comment by WindSoilder** [comment]
@abusch Hi, sorry for the conflict, would you please resolve it first?

**Comment by abusch** [comment]
Done!

**Pull request merged**
